sdk/storage/azure-storage-blob/tests/test_container.py

```diff
@@ -1772,7 +2090,10 @@ def test_standard_blob_tier_set_tiers_with_sas(self, storage_account_name, stora
     # once we have premium tests, still we don't want to test Py 2.7
     # @pytest.mark.skipif(sys.version_info < (3, 0), reason="Batch not supported on Python 2.7")
     @BlobPreparer()
-    def test_premium_tier_set_tier_api_batch(self, storage_account_name, storage_account_key):
+    def test_premium_tier_set_tier_api_batch(self, **kwargs):
```

Comment:
Did you try anything with this test? Looks like we need a Premium Page Blob account that supports batch? That just may not be a thing :)

Reply:
Yeah I have no idea what "premium" means in this context 😆. I remember trying a bunch of different things when initially triaging this in our 'triage all skipped tests' item from a couple weeks / month(?) ago, so just left it skipped for now (it's in the backlog!! 🤣)